infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -464,7 +464,10 @@ class PublicationDao(
                   postgis.st_x(candidate_operational_point.location) as point_x,
                   postgis.st_y(candidate_operational_point.location) as point_y
                 from layout.operational_point_version_view candidate_operational_point
-                where candidate_operational_point.expiry_time is null
```

Comment:
Niin tuonhan olisi tosiaan saanut myös `expiry_time is null and deleted=false` ilman selectiä päätauluun :thinking: